securedrop/source_app/session_manager.py

```diff
@@ -27,80 +28,76 @@ class UserHasBeenDeleted(_InvalidUserSession):
 
 
 class SessionManager:
-    _CACHE = SimpleCache()  # A mapping of passphrase -> SourceUser
```

Comment:
I have removed the local cache here to address session timeout bugs discussed in #5694.